modules/oauth/src/main/java/org/mule/extension/oauth2/internal/OAuthConstants.java

```diff
@@ -26,11 +26,11 @@
   String GRANT_TYPE_REFRESH_TOKEN = "refresh_token";
 
   // Expressions to extract parameters from standard token url response.
-  String ACCESS_TOKEN_EXPRESSION = "#[regex('" + ".*\"access_token\"[ ]*:[ ]*\"([^\\\"]*)\".*" + "')]";
-  String REFRESH_TOKEN_EXPRESSION = "#[regex('" + ".*\"refresh_token\"[ ]*:[ ]*\"([^\\\"]*)\".*" + "')]";
-  String EXPIRATION_TIME_EXPRESSION = "#[regex('" + ".*\"expires_in\"[ ]*:[ ]*([\\\\d]*).*" + "')]";
+  String ACCESS_TOKEN_EXPRESSION = "#[mel:regex('" + ".*\"access_token\"[ ]*:[ ]*\"([^\\\"]*)\".*" + "')]";
```

Comment:
Regex lives in core so it's easier to maintain but yes, we need to see whether it lives on, specially since dw offers the same I believe.